setup.py

```diff
@@ -24,7 +24,7 @@ def read(*parts):
     url='https://github.com/python-babel/django-babel/',
     packages=find_packages(exclude=('tests',)),
     install_requires=[
-        'django>=1.4,<1.9',
+        'django>=1.4,<1.10',
```

Comment:
interesting, I'd like to be able to express which versions are valid here. Raise a PR with that fix